types/jws/index.d.ts

```diff
@@ -148,7 +148,27 @@ export type Algorithm = 'HS256' | 'HS384' | 'HS512' | 'RS256' |
                         'ES512' | 'PS256' | 'PS384' | 'PS512' |
                         'none';
 
-export interface Header {
+export type Header = {
     alg: Algorithm;
-    [name: string]: string;
+    jwk?: JWK;
+    typ?: string;
+    cty?: string;
+    crit?: ReadonlyArray<string>;
+} & CertificateProperties & {
```

Comment:
Instead of using intersection type here, i would recommend use `export interface Header extends CertificateProperties` instead